sites/partners/styles/overrides.scss

```diff
@@ -0,0 +1,27 @@
+// The header logo should allow for a larger maximum height than the default
+// logo.
+.navbar-brand .logo .logo__image {
```

Comment:
I was about to suggest using Tailwind's min-height classes so we don't have some styles that are tailwind and some that aren't, but it seems like we'd have to add custom min-height anyway:
https://tailwindcss.com/docs/min-height
They have some for max-height though:
https://tailwindcss.com/docs/max-height

So up to you if you want to bother trying to get it to work that way

Reply:
I'll keep this as is right now, but will reach out to you directly to make sure I'm understanding your suggestion.